### ISSUE TITLE
Problem on DigitalSignature

### ISSUE THREAD

**Issue by gmile** [edenlabllc/ehealth.api#2060]
From logs:
```
** (Jason.EncodeError) invalid byte 0xFE in <<254, 237, 254, 237, 0, 0, 0, 2, 0, 0, 0, 1, 0, 0, 0, 1, 0, 18, 112, 98, 95, 115, 105, 103, 110, 95, 49, 57, 48, 48, 51, 50, 49, 49, 48, 54, 0, 0, 0, 0, 90, 177, 9, 47, 0, 0, 3, 32, 48, 130, ...>>
    (jason) lib/jason.ex:137: Jason.encode_to_iodata!/2
    (phoenix) lib/phoenix/controller.ex:740: Phoenix.Controller.do_render/4
    (digital_signature_api) lib/digital_signature_api/web/controllers/digital_signatures_controller.ex:1: DigitalSignature.Web.DigitalSignaturesController.action/2
    (digital_signature_api) lib/digital_signature_api/web/controllers/digital_signatures_controller.ex:1: DigitalSignature.Web.DigitalSignaturesController.phoenix_controller_pipeline/2
    (digital_signature_api) lib/digital_signature_api/web/endpoint.ex:1: DigitalSignature.Web.Endpoint.instrument/4
    (phoenix) lib/phoenix/router.ex:278: Phoenix.Router.__call__/1
    (digital_signature_api) lib/plug/error_handler.ex:64: DigitalSignature.Web.Router.call/2
    (digital_signature_api) lib/digital_signature_api/web/endpoint.ex:1: DigitalSignature.Web.Endpoint.plug_builder_call/2
```

**Comment by gmile** [comment]
@ybod looks like we cannot render response in a controller? How can that be? Could it be that we've received a bad answer from OSCP and are tying to JSON-encode it during response? (this is a wild guest but I don't have any other idea at the moment)

**Comment by ybod** [comment]
No, the answer from OCSP is used internally by the library to check OCSP validity and not encoded to JSON.

**Comment by ybod** [comment]
Should be fixed in `1.48.0`. It will return correct error: `422 Malformed Request`.
